Sources/CardPayments/APIRequests/GetOrderInfoResponse.swift

```diff
@@ -21,3 +21,9 @@ struct Amount: Codable {
     let currencyCode: String
     let value: String
 }
+
+struct Link: Decodable {
```

Comment:
Don't think we need to move `Link` from  `ConfirmPaymentSourceResponse.swift` anymore